docs/userguides/accounts.md

```diff
@@ -55,7 +55,39 @@ For example, you can [generate](../commands/accounts.html#accounts-generate) an
 ape accounts generate <ALIAS>
 ```
 
-It will prompt you for a passphrase.
+Ape will prompt you for entropy which is used to increase randomness when creating your account.
+
+Ape will then prompt you whether you want to show your mnemonic.
+
+If you do not want to see your mnemonic you can select `n` or alternatively you can use the `--hide-mnemonic` option.
+
+```bash
+ape accounts generate <ALIAS> --hide-mnemonic
+```
+
+If you elected to show your mnemonic Ape will then show you your newly generated mnemonic.
+
+Ape will then prompt you for a passphrase which you will need to enter twice to confirm.
```

Comment:
Maybe add "This passphrase is used to encrypt your account on disk, for extra security. You will be prompted for it each time you load your account, so make sure to remember it."